lib/rubocop/cop/rspec/factory_bot/static_attribute_defined_dynamically.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Prefer declaring static attribute values without a block.
+        #
+        # @example
+        #   # bad
+        #   kind { :static }
+        #
+        #   # good
+        #   kind :static
+        #
+        #   # bad
+        #   comments_count { 0 }
+        #
+        #   # good
+        #   comments_count 0
+        #
+        #   # bad
+        #   type { User::MAGIC }
+        #
+        #   # good
+        #   type User::MAGIC
+        class StaticAttributeDefinedDynamically < Cop
+          MSG = 'Dont use a block to set a static value to an attribute.'.freeze
+
+          def_node_matcher :block_ast_method, <<-PATTERN
+          (block (send nil? _) _ ($_ ...))
```

Comment:
Instead of `$_` you can be explicit that you don't want a `send` using negation `$!send`:

```
(block (send nil? _) _ ($!send ...))
```

I didn't test it locally but it should work 😄

Reply:
Yes, works great! I removed the matcher as we shouldn't need it anymore 👍